casper/src/test/resources/logback-test.xml

```diff
@@ -1,18 +1,19 @@
 <configuration>
 
-  <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener"/>
```

Comment:
this reformatting seems wrong

Reply:
only because it's been taken out of context :) Unless you're arguing for 2 spaces of indent for XMLs? scalafmt doesn't yell for sure, so I guess it's up for discussion?